synapse/handlers/space_summary.py

```diff
@@ -32,7 +32,6 @@
 logger = logging.getLogger(__name__)
 
 # number of rooms to return. We'll stop once we hit this limit.
-# TODO: allow clients to reduce this with a request param.
```

Comment:
The MSC has no mention of this, so I don't think mentioning it here makes sense.